service/history/transferQueueStandbyTaskExecutor.go

```diff
@@ -408,12 +408,41 @@ func (t *transferQueueStandbyTaskExecutor) processStartChildExecution(
 			return nil, nil
 		}
 
-		if childWorkflowInfo.StartedId != common.EmptyEventID {
+		workflowClosed := !mutableState.IsWorkflowExecutionRunning()
+		childStarted := childWorkflowInfo.StartedId != common.EmptyEventID
+		childAbandon := childWorkflowInfo.ParentClosePolicy == enumspb.PARENT_CLOSE_POLICY_ABANDON
+
+		if workflowClosed && !(childStarted && childAbandon) {
```

Comment:
If condition workflowClosed && !childStarted && childAbandon, we cannot go ahead here to start child. 
It is desired behavior, but due to technical reason we cannot do that. 
Can you put some comment here explain this situation.